src/som/interpreter/actors/Actor.java

```diff
@@ -75,6 +77,9 @@ public static Actor createActor(final VM vm) {
   /** Is scheduled on the pool, and executes messages to this actor. */
   protected final ExecAllMessages executor;
 
+  /** Flag that indicates if a step-to-next-turn action has been made in the previous message. */
+  protected boolean stepToNextTurn;
```

Comment:
why? but then the user will need to have enabled tracing always. Is tracing a functionality optional or mandatory to debug? I thought it was optional